template/src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+/**
```

Comment:
I think we shouldn’t ship this by default. It’s a little bit advanced feature, and it’s fine to just document it. Can you add relevant documentation section to `template/README.md` instead?